drivers/usb/device/usb_dc_sam0.c

```diff
@@ -234,7 +242,9 @@ int usb_dc_attach(void)
 	(void)memset(data->descriptors, 0, sizeof(data->descriptors));
 	regs->DESCADD.reg = (uintptr_t)&data->descriptors[0];
 
-	regs->INTENSET.reg = USB_DEVICE_INTENSET_EORST;
+	regs->INTENSET.reg = USB_DEVICE_INTENSET_EORST |
+			     USB_DEVICE_INTENSET_EORSM |
+			     USB_DEVICE_INTENSET_SUSPEND;
```

Comment:
FYI, this is not how ASF handles EORSM and SUSPEND (these interrupts are enabled on-demand).

See #26657

Also ASF implementation:
https://github.com/adafruit/asf4/blob/d270f79aa16dd8fd4ae3b6c14544283dcb992e9c/samd51/hpl/usb/hpl_usb.c#L982-L984

https://github.com/adafruit/asf4/blob/d270f79aa16dd8fd4ae3b6c14544283dcb992e9c/samd51/hpl/usb/hpl_usb.c#L992-L994